src/Payments/Api.php

```diff
@@ -80,8 +80,12 @@ public function add_payment_method_script_dependencies( $dependencies, $handle )
 	 * Add payment method data to Asset Registry.
 	 */
 	public function add_payment_method_script_data() {
-		$script_data = $this->payment_method_registry->get_all_registered_script_data();
+		// Enqueue the order of enabled gateways as `paymentGatewaySortOrder`.
+		$available_gateways = WC()->payment_gateways->get_available_payment_gateways();
+		$this->asset_registry->add( 'paymentGatewaySortOrder', array_keys( $available_gateways ) );
```

Comment:
This should be wrapped in a check for whether this is already registered:

```suggestion
		if ( ! $this->asset_registry->exists( 'paymentGatewaySortOrder' ) ) {
			$this->asset_registry->add( 'paymentGatewaySortOrder', array_keys( $available_gateways ) );
		}
```

Otherwise you can get a thrown exception if the key already exists (which happened to me while trying to test).

Reply:
Interesting, I'll add the check. Curious why this code would be firing multiple times though - is this expected?

Reply:
Ah I see the other assets are protected by a check, missed that. Added check here: 16139feb4f37a8cd30ed164c38bbf3129b2c7ea8